tfrecorder/beam_pipeline.py

```diff
@@ -76,6 +66,7 @@ def _get_pipeline_options(
     job_dir: str,
     project: str,
     region: str,
+    tfrecorder_wheel:str,
```

Comment:
nit: ":" -> ": "

Reply:
Thx, linter didn't find that for some reason.